wolfcrypt/src/rsa.c

```diff
@@ -2130,35 +2397,87 @@ int wc_MakeRsaKey(RsaKey* key, int size, long e, WC_RNG* rng)
     }
 #endif
 
-    if ((err = mp_init_multi(&p, &q, &tmp1, &tmp2, &tmp3, NULL)) != MP_OKAY)
-        return err;
+    err = mp_init_multi(&p, &q, &tmp1, &tmp2, &tmp3, NULL);
 
-    err = mp_set_int(&tmp3, e);
+    if (err == MP_OKAY)
+        err = mp_set_int(&tmp3, e);
+
+    failCount = 5 * (size / 2);
```

Comment:
Can you add a comment that indicates why we are multiplying by 5 here?

Reply:
ACK. Adding comment that it comes from the spec.